glib/src/subclass/mod.rs

```diff
@@ -86,9 +87,6 @@
 //!         type Instance = subclass::simple::InstanceStruct<Self>;
 //!         type Class = subclass::simple::ClassStruct<Self>;
```

Comment:
```suggestion
```
Those can be omitted now, no?

Reply:
Probably. I guess the documentation should cover the way they have default values and how they can be set to something else, but they can probably be omitted in a simple example.

Reply:
Yeah, that's something for the docs of the macro. Describing what it does and what the "defaults" are that it's going to use